subsys/bluetooth/services/ots/ots_client.c

```diff
@@ -133,7 +138,28 @@ static void tx_done(struct bt_gatt_ots_l2cap *l2cap_ctx,
 	/* Not doing any writes yet */
 	BT_ERR("Unexpected call, context: %p, conn: %p", l2cap_ctx, (void *)conn);
 }
+static void write_obj_tx_done(struct bt_gatt_ots_l2cap *l2cap_ctx,
+		    struct bt_conn *conn)
+{
+	int err;
+	size_t written;
+
+	BT_DBG("%s call, context: %p, conn: %p", __func__, l2cap_ctx, (void *)conn);
+	written = cur_inst->sent_size;
+	BT_WARN("written %d", written);
+	BT_WARN("written %d", l2cap_ctx->tx.len_sent);
+	BT_WARN("written %d", cur_inst->l2cap_ctx.tx.len_sent);
+	err = bt_gatt_ots_l2cap_disconnect(l2cap_ctx);
```

Comment:
those look like debug/wip statements. Also BT_DBG (and LOG_DBG) automatically include the function name. Suggestion:
```c 
	int err;
	size_t written;

	written = cur_inst->sent_size;

	BT_DBG("ctx: %p, conn: %p, written: %d",
	       l2cap_ctx, (void *)conn, written);

	err = bt_gatt_ots_l2cap_disconnect(l2cap_ctx);
```

Reply:
Revised.